scripts/utils.js

```diff
@@ -41,6 +48,22 @@ export async function promptContinue( msg ) {
 	return answer.continue;
 }
 
+export async function abortAndSwitchToBranch( msg, type, branchName = null ) {
+	if ( type === 'error' ) {
+		error( msg );
+	} else if ( type === 'warning' ) {
+		warning( msg );
+	} else {
+		info( msg );
+	}
```

Comment:
Minor nit: we could also use `NOTICE_LEVEL` in this code.
```suggestion
	if ( type === NOTICE_LEVEL.INFO ) {
		error( msg );
	} else if ( type === NOTICE_LEVEL.WARNING ) {
		warning( msg );
	} else {
		info( msg );
	}
```